rj_drawing_msgs/msg/DrawPose.msg

```diff
@@ -0,0 +1 @@
+rj_geometry_msgs/Pose pose
```

Comment:
`DrawColor color`?

Reply:
We don't have a way of rendering poses yet but I was planning on drawing them with just red/blue axes in the standard format. Any thoughts on a better way that could represent colors?